sql/updates/auth/2015_09_23_00_auth.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `account` DROP COLUMN `mutetime`, DROP COLUMN `mutereason`, DROP COLUMN `muteby`; 
```

Comment:
These changes need to be reflected in the `auth_database.sql` file (use the `mysqldump command and then reset not required changes, like the changing mysql version in the file).